packages/seacas/libraries/ioss/src/faodel/CMakeLists.txt

```diff
@@ -0,0 +1,87 @@
+# @HEADER
+# ************************************************************************
```

Comment:
Wrong copyright notice and wrong date.

Reply:
I removed the copyright notice to match the CMakeLists.txt files of the other DatabaseIO plugins (commit e297a3282248685453f21807a0cc4638c8ff9be3).